src/main/antlr4/org/apache/spark/sql/ml/parser/RikaiExtSqlBase.g4

```diff
@@ -153,6 +153,10 @@ fragment DIGIT
     : [0-9]
     ;
 
+fragment NONDIGIT
```

Comment:
the sql parser is case insensitive.  

https://github.com/eto-ai/rikai/blob/master/src/main/scala/ai/eto/rikai/sql/spark/parser/RikaiExtSparkSQLParser.scala#L86

Maybe just make `IDENTIFER` to be:

```
IDENTFIER：(LETTER | '_') (LETTER|DIGIT| '_')* ;
```